textdb/textdb-dataflow/src/test/java/edu/uci/ics/textdb/dataflow/neextractor/NEExtractorTestConstants.java

```diff
@@ -0,0 +1,121 @@
+package edu.uci.ics.textdb.dataflow.neextractor;
+
+import edu.uci.ics.textdb.api.common.*;
+import edu.uci.ics.textdb.common.constants.SchemaConstants;
+import edu.uci.ics.textdb.common.field.*;
+
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Created by Sam on 16/4/27.
+ */
+public class NEExtractorTestConstants {
```

Comment:
Please add code using TestUtils.containsAllResults(), even though it may fail.  At least you need to know how to write that piece of code.